atomic_reactor/plugins/store_metadata.py

```diff
@@ -151,17 +148,16 @@ def run(self):
             media_types += media_results
 
         if media_types:
-            annotations['media-types'] = json.dumps(sorted(list(set(media_types))))
+            annotations['media-types'] = sorted(list(set(media_types)))
 
         self.apply_plugin_annotations(annotations)
 
-        try:
-            self.workflow.osbs.update_annotations_on_build(pipeline_run_name, annotations)
-            self.log.debug("annotations on pipeline run '%s' were updated", pipeline_run_name)
-            self.log.debug("annotations on pipeline run are : '%s'",
-                           self.workflow.osbs.get_build_annotations(pipeline_run_name))
-        except OsbsResponseException:
-            self.log.debug("annotations: %r", annotations)
-            raise
+        if self.workflow.annotations_result:
+            annotations_result = {'plugins-metadata': annotations['plugins-metadata']}
```

Comment:
How big is plugins-metadata after converting to JSON? Do we need to worry about hitting the 4096 byte limit of Tekton results?

Reply:
it is working fine now with full build

Reply:
And what is the size? Are we barely getting by, in danger of breaking any time? Or do we have lots of extra space?

Reply:
I'll have to check

Reply:
currently with all plugins completed it is 3400